src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-    version="3.0">
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee//web-app_3_1.xsd"
```

Comment:
- [ ] typo `javaee//web-app_3_1.xsd` -> `javaee/web-app_3_1.xsd` ?

Reply:
No need to revise as the pointed out part was deleted.
https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/455/commits/03f53f1bd3d760d2eeceaab9cb1e73e48c3f17d5